release-notes/elastic-security/deprecations.md

```diff
@@ -18,6 +18,10 @@ Review the deprecated functionality for {{elastic-sec}}. While deprecations have
 
 ## 9.0.0 [elastic-security-900-deprecations]
 
+::::{dropdown} Removes Defend for Containers (D4C)
+Defend for containers is no longer supported starting with {{stack}} 9.0. 
```

Comment:
We have title case and sentence case. Is it `Defend for Containers` or `Defend for containers`?